doc/instrumentation.rst

```diff
@@ -269,4 +269,468 @@ Available Instruments
 
 This section lists instruments that are currently part of devlib.
 
-TODO
+.. todo:: Add other instruments
+
+
+Baylibre ACME BeagleBone Black Cape
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _official project page:            http://baylibre.com/acme/
+.. _image built for using the ACME:   https://gitlab.com/baylibre-acme/ACME-Software-Release/blob/master/README.md
+.. _libiio (the Linux IIO interface): https://github.com/analogdevicesinc/libiio
+.. _Linux Industrial I/O Subsystem:   https://wiki.analog.com/software/linux/docs/iio/iio
+.. _Texas Instruments INA226:         http://www.ti.com/lit/ds/symlink/ina226.pdf
+
+From the `official project page`_:
+
+  [The Baylibre Another Cute Measurement Equipment (ACME)] is an extension for the BeagleBone Black (the ACME Cape), designed to provide multi-channel power and temperature measurements capabilities to the BeagleBone Black (BBB). It comes with power and temperature probes integrating a power switch (the ACME Probes), turning it into an advanced all-in-one power/temperature measurement solution.
+
+  The ACME initiative is completely open source, from HW to SW drivers and applications.
+
+
+The Infrastructure
+^^^^^^^^^^^^^^^^^^
+
+Retrieving measurement from the ACME through devlib requires:
+
+  - a BBB running the `image built for using the ACME`_ (micro SD card required);
+  - an ACME cape on top of the BBB;
+  - at least one ACME probe [#acme_probe_variants]_ connected to the ACME cape;
+  - a BBB-host interface (typically USB or Ethernet) [#acme_name_conflicts]_;
+  - a host (the one running devlib) with `libiio (the Linux IIO interface)`_ installed, and a Python environment able to find the libiio Python wrapper *i.e.* able to ``import iio`` as communications between the BBB and the host rely on the `Linux Industrial I/O Subsystem`_ (IIO).
```

Comment:
Can you better detail how to get the python IIO support?
It seems that, on my Ubuntu 16.04, I cannot just pip install it...

Reply:
Ok, the support seems to be provided by the python-libiio package... maybe we can add an example note for Ubuntu systems?

Reply:
To install it, I downloaded the [archive provided](http://swdownloads.analog.com/cse/travis_builds/master_latest_libiio-trusty.tar.gz) which contains the Python files in `usr/lib/python2.7/site-packages` which I copied to the corresponding location and made sure that my path (in the Python environment) was accessing that directory. This is probably not the cleanest way and seemed a bit counter-intuitive.

If there is an APT package then it's much better! Also, I couldn't find an module on PyPI for libiio (which would allow the better option of using pip) ...